src/instructlab/data/generate_data.py

```diff
@@ -51,7 +147,7 @@ def gen_data(
         from instructlab.model.backends import backends
         from instructlab.model.backends.llama_cpp import Server as llama_cpp_server
 
-        backend_instance = backends.select_backend(cfg=serve_cfg, model_path=model_path)
+        backend_instance = backends.select_backend(cfg=serve_cfg, model_path=model_name)
```

Comment:
Arg rename?

Reply:
I renamed some arg so they could be used as kwargs fed directly into SDG. `generate_data` required model_name, so I just switched it for us as well. 

this isn't user-facing.

Reply:
Okay - it's a little hard as a reviewer to know what arg renaming is purposeful and why it's being done versus knowing what's a mistake.

Refactors ideally should be done prior to new features - for example, I did this here prior to OCI downloads so people could focus their attention on what was being done and why: https://github.com/instructlab/instructlab/pull/2808

Something to consider for the future.

Reply:
right, it kind of had to be done here due to implications of passing things as **kwargs. I can add a comment in the code as to why this exists the way it does now if that would help.

this is internal code though, nothing user facing, an implementation detail

Reply:
No worries! Will resolve this thread - thanks for the explanation!